docs/advanced/document-store/benchmark.md

```diff
@@ -10,18 +9,18 @@
 :scale: 0 %
 ```
 
-We create a DocumentArray with one million Documents using randomly generated data from a uniform distribution and benchmark all supported document stores.
+We create a DocumentArray with one million Documents based on [SIFT1M](https://www.tensorflow.org/datasets/catalog/sift1m), a dataset containing 1 million objects of 128d and using l2 distance metrics, and benchmark the following document stores.
```

Comment:
I'm not sure whether the link from tensorflow is the official sift1m page I think the home page is http://corpus-texmex.irisa.fr/